File: Python/242. Valid Anagram.py
class Solution:
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        d1, d2 = dict(), dict()
        for c in s:
            d1[c] = d1.get(c, 0) + 1
        for c in t:
            d2[c] = d2.get(c, 0) + 1
        return d1 == d2

File: C++/242. Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int len1 = s.length();
        int len2 = t.length();
        if(len1 != len2)
            return false;
        unordered_map<int,int> maple;
        for(int i=0;i<len1;i++)
        {
            maple[s[i]]++;
            maple[t[i]]--;
        }
        for(auto cnt: maple)
        {
            if(cnt.second != 0)
                return false;
        }
        return true;
    }
};